scripts/qa_crowbarsetup.sh

```diff
@@ -4169,9 +4176,26 @@ puts y.to_yaml" > /root/keystone-test-pw-reset.yaml
     safely crowbar batch --timeout 900 build < /root/keystone-test-pw-reset.yaml
 }
 
+function onadmin_setupsalt
+{
+    test -e /opt/dell/crowbar_framework/barclamps/salt.yml || return 0
+    add_salt_repo
+    do_one_proposal salt default
+}
+
+function onadmin_testsalt
+{
+    test -e /opt/dell/crowbar_framework/barclamps/salt.yml || return 0
```

Comment:
Code duplicatop. Sure its only one line, but this check looks like it should be a function.